src/View/UIViewTrait.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace BootstrapUI\View;
+
+/**
+ * UIViewTrait: Trait that aims to:
+ *
+ * loading the custom UIBootstrap helpers.
+ * setting the View layout to the UIBootstrap's one.
+ */
+trait UIViewTrait
+{
+    /**
+     * Initialization hook method.
+     *
+     * @return void
+     */
+    public function initialize()
```

Comment:
I wonder if this method should be named something else and the initialize() should be in the UIView instead which calls this method. @lorenzo Thoughts?